### ISSUE TITLE
GSoC 2016 Location autocomplete

### ISSUE THREAD

**Issue by bphenriques** [systers/malaria-app-ios#21]
In planmytrip, add an autocomplete location list:

The app will be usually in offline mode so, this feature makes more sense if the user is connected to the internet unless is possible to have the information offline. 

E.g. 
https://github.com/watsonbox/ios_google_places_autocomplete
https://maniacdev.com/2013/10/ios-completion-providing-search-with-autocompletion-and-map-marking-using-google-places

**Comment by manas15** [comment]
I have implemented Location AutoComplete functionality using Google Place Autocomplete. 
When the user is connected to internet she can use this by clicking the "search over web button " and a new screen will come where she will get suggestions as she types.
And once she makes a selection the selected data will come on the Destination Field.

**Comment by manas15** [comment]
For the implementation Google Places API has been used. 
For the documentation go ahead and read from : https://developers.google.com/places/ios-api/start#step-1-get-the-latest-version-of-xcode

*Note: To implement this we have to generate an api key from the google developers console. For the same I have created a project by the name Systers-malaria-app-ios. And corresponding to this app on console I have registered the following bundle identifiers:
1. anita-borg.malaria-ios
2. org.systers.peacecorps.malaria 
Only these bundle identifiers will be authenticated by google places api to receive response from the api.
